index.js

```diff
@@ -1,20 +1,20 @@
 'use strict';
 const net = require('net');
 
-const getPort = port => new Promise((resolve, reject) => {
+const getPort = (port, host) => new Promise((resolve, reject) => {
 	const server = net.createServer();
 
 	server.unref();
 	server.on('error', reject);
 
-	server.listen(port, () => {
+	server.listen({port, host}, () => {
```

Comment:
It also has an overload in which accepts an object according to the [docs](https://nodejs.org/api/net.html#net_server_listen_options_callback). I think we can go with one or another.